Source/WebCore/bindings/js/SerializedScriptValue.cpp

```diff
@@ -1390,6 +1393,24 @@ class CloneSerializer : CloneBase {
                 write(String::fromLatin1(JSC::Yarr::flagsString(regExp->regExp()->flags()).data()));
                 return true;
             }
+            if (auto* errorInstance = jsDynamicCast<ErrorInstance*>(obj)) {
```

Comment:
According to the spec text, (https://html.spec.whatwg.org/multipage/structured-data.html#structuredserializeinternal), we need to obtain `name` via `get` (this is user observable behavior. User can define `name` getter), and check the name via specified one. And after that, we need to run `[[GetOwnProperty]]` to obtain `"message"` and need to check whether message has data property. And it seems that this code is not aligned to the spec's behavior. Can you double-check the spec behavior since they are user observable?

Reply:
> and need to check whether message has data property
That part works and is covered by WPT tests. I know because I got it wrong in my first iteration.

Anyway, I was not aware of the spec so I am not surprised if the implementation doesn't match it closely. I'll look at the spec and align more.